.github/workflows/readme-check.yml

```diff
@@ -0,0 +1,26 @@
+name: Check README Updates
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  workflow_dispatch:
+  merge_group:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check-readme:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
Please use `actions-rust-lang/setup-rust-toolchain` to install rust. Also please install the same rust version that other jobs use. (You can check the example [here](https://github.com/paritytech/polkadot-sdk/blob/225536c9f57c9ff4425aa94e275648ce334eb4cc/.github/workflows/tests-misc.yml#L350-L356))